sklearn/decomposition/kernel_pca.py

```diff
@@ -67,6 +77,12 @@ class KernelPCA(BaseEstimator, TransformerMixin):
         Maximum number of iterations for arpack.
         If None, optimal value will be chosen by arpack.
 
+    iterated_power : int >= 0, or 'auto', (default 'auto')
+        Number of iterations for the power method computed by
+        svd_solver == 'randomized'.
+
+        .. versionadded:: 0.20
```

Comment:
Not going to make it into 0.20. Mark as 0.21

Reply:
ok